aklite-callback.sh

```diff
@@ -4,6 +4,23 @@
 # It notifies the fiotest container about the callback so that it can
 # handle everything.
 
+# First make sure this app is still installed:
+# aktualizr-lite "uninstalls" a compose-app by basically running:
+#  docker-compose stop; rm -rf /var/sota/compose-apps/fiotest
+# we can detect we were deleted by looking for that directory:
+cd $(dirname $0)
+if [ ! -d ./compose-apps/fiotest ] ; then
+	if [ -f /etc/sota/conf.d/z-90-fiotest.toml ] ; then
+		echo "fiotest appears to be removed. disabling callback"
+		rm /etc/sota/conf.d/z-90-fiotest.toml
+	fi
+
+	# this gets tricky: we need to restart aklite. We don't want to do
+	# it when aklite is performing an update. To be safe we just exit
+	# now and eventually aklite will restart and stop calling us.
+	exit 0
```

Comment:
this will still leave the alite-callback.sh file here unless you want to do an rm just before the exit.

Reply:
if we remove ourselve, then aklite will log errors about being unable to execute the callback until its restarted.

Reply:
so aklite is caching the callbacks instead of parsing every time? how expensive would it be to just go through every toml file?

Reply:
you mean have aklite dynamically re-read its configuration? that's probably non-trivial